src/storage/src/hummock/file_cache/file.rs

```diff
@@ -274,6 +274,7 @@ impl CacheFile {
 #[cfg(test)]
 mod tests {
     use super::*;
+    use crate::hummock::file_cache::test_utils::tempdir;
```

Comment:
Seems these storage changes are included by accident?

Reply:
I noticed that the unit tests in `cache.rs` were using a `tempdir` function that would use `/risingwave` for temp directories if the env var `RISINGWAVE_CI`  was set. Since, this is using a temp directory to store the same cache data files as the unit tests in `cache.rs`, I thought it would make sense to update this code to use the same `tempdir` function as `cache.rs`.

I can undo this change if desired.